wiki/en/Software-Manual.md

```diff
@@ -240,9 +240,9 @@ rate is not higher than your available internet upload speed (check this with a
 
 <figure><img src="{% include img/en-screenshots/settings-advanced.inc %}" style="border: 5px solid grey;" loading="lazy" alt="Image of advanced settings window"></figure>
 
-### Custom directory server address
+### Custom Directory server address
```

Comment:
![image](https://user-images.githubusercontent.com/1549463/182020986-ad9caab4-847c-47eb-b5ce-3b0fd653e5e4.png)
"Custom Directories" (as it can now contain many).